Documentation/k8s-pre-reqs.md

```diff
@@ -167,10 +160,51 @@ imagePullSecrets:                # here are the new
 
 After doing this for all service accounts all pods should be able to pull the image from your registry.
 
-## Using Rook in Kubernetes
+## Bootstrapping Kubernetes
+
+Rook will run wherever Kubernetes is running. Here are some simple environments to help you get started with Rook.
+
+### Minikube
+
+To install `minikube`, refer to this [page](https://github.com/kubernetes/minikube/releases). Once you have `minikube` installed, start a cluster by doing the following:
+
+```console
+$ minikube start
+Starting local Kubernetes cluster...
+Starting VM...
+SSH-ing files into VM...
+Setting up certs...
+Starting cluster components...
+Connecting to cluster...
+Setting up kubeconfig...
+Kubectl is now configured to use the cluster.
+```
+
+After these steps, your minikube cluster is ready to install Rook on.
+
+### Kubeadm
+
+You can easily spin up Rook on top of a `kubeadm` cluster.
+You can find the instructions on how to install kubeadm in the [Install `kubeadm`](https://kubernetes.io/docs/setup/independent/install-kubeadm/) page.
+
+By using `kubeadm`, you can use Rook in just a few minutes!
 
-Now that you have a Kubernetes cluster running, you can start using Rook with [these steps](ceph-quickstart.md#deploy-rook).
+### New local Kubernetes cluster with Vagrant
 
-## Using Rook on Tectonic Bare Metal
+For a quick start with a new local cluster, use the Rook fork of [coreos-kubernetes](https://github.com/rook/coreos-kubernetes). This will bring up a multi-node Kubernetes cluster with `vagrant` and CoreOS virtual machines ready to use Rook immediately.
 
-Follow [these instructions](tectonic.md) to run Rook on Tectonic Kubernetes
+```console
+git clone https://github.com/rook/coreos-kubernetes.git
```

Comment:
https://github.com/rook/coreos-kubernetes project seems to still use K8S v1.7.3 (?), shouldn't we point to another project for virtual local Kubernetes?